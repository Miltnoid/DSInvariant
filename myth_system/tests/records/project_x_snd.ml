type nat =
  | O
  | S of nat

let getXsnd : {x:(nat * nat)} -> nat |>
  { {x = (0,1)} => 1
  | {x = (1,0)} => 0 } = ?
