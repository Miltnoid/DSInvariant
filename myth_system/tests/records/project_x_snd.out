(* Generated Tue Mar  3 15:20:26 2015 *)
let getXsnd : { x : (nat * nat) } -> nat =
  fun (r1:{ x : (nat * nat) }) -> #2 (r1.x)
;;
